test/spec/update-listeners.js

```diff
@@ -105,7 +105,7 @@ describe('update-listeners', () => {
   }
 
   const createTestDBs = async() => {
-    await testUtils.createTestDBs(['test_db1' + suffix, 'test_db3' + suffix])
+    await testUtils.createTestDBs(['test_db1' + suffix, 'test_db3' + suffix, 'account/1/2'])
```

Comment:
I don't think this change is needed for your test. Please remove it

Reply:
Sorry, forgot about that line. I removed it.